impeller/entity/contents/content_context.cc

```diff
@@ -352,11 +231,21 @@ std::shared_ptr<Texture> ContentContext::MakeSubpass(
   if (context->GetCapabilities()->SupportsOffscreenMSAA() && msaa_enabled) {
     subpass_target = RenderTarget::CreateOffscreenMSAA(
         *context, texture_size, SPrintF("%s Offscreen", label.c_str()),
-        RenderTarget::kDefaultColorAttachmentConfigMSAA, std::nullopt);
+        RenderTarget::kDefaultColorAttachmentConfigMSAA  //
+#ifndef FML_OS_ANDROID
```

Comment:
Optimizing out the stencil forces a new variant on Vulkan. Perhaps there is a faster way to attach a stencil and not use it, but this seems reasonable?